client/extensions/woocommerce/state/ui/payments/methods/README.md

```diff
@@ -0,0 +1,58 @@
+Payment Methods
+================
+
+This module is used to manage payment method related UI.
+
+## Actions
+
+### `openPaymentMethodForEdit( siteId: number, id: string )`
```

Comment:
I was following the format in calypso, but i'm open to a chat about a different way to display our documentation in the readme. I do think we should keep the reference here, though — for one thing, this auto-populates into devdocs, so you can [view the readmes](https://wpcalypso.wordpress.com/devdocs/client/extensions/woocommerce/state/sites/settings/general/README.md), or [search for a selector](https://wpcalypso.wordpress.com/devdocs?term=areSettingsGeneralLoaded).